src/Shared/SemanticConventions.cs

```diff
@@ -7,116 +7,23 @@ namespace OpenTelemetry.Trace;
-    public const string AttributeMessagingTempDestination = "messaging.temp_destination";
-    public const string AttributeMessagingProtocol = "messaging.protocol";
-    public const string AttributeMessagingProtocolVersion = "messaging.protocol_version";
-    public const string AttributeMessagingUrl = "messaging.url";
-    public const string AttributeMessagingMessageId = "messaging.message_id";
-    public const string AttributeMessagingConversationId = "messaging.conversation_id";
-    public const string AttributeMessagingPayloadSize = "messaging.message_payload_size_bytes";
-    public const string AttributeMessagingPayloadCompressedSize = "messaging.message_payload_compressed_size_bytes";
-    public const string AttributeMessagingOperation = "messaging.operation";
 
     public const string AttributeExceptionEventName = "exception";
     public const string AttributeExceptionType = "exception.type";
     public const string AttributeExceptionMessage = "exception.message";
     public const string AttributeExceptionStacktrace = "exception.stacktrace";
-    public const string AttributeErrorType = "error.type";
-
-    // v1.21.0
-    // https://github.com/open-telemetry/semantic-conventions/blob/v1.21.0/docs/database/database-spans.md
-    public const string AttributeServerSocketAddress = "server.socket.address"; // replaces: "net.peer.ip" (AttributeNetPeerIp)
-
-    // v1.23.0
-    // https://github.com/open-telemetry/semantic-conventions/blob/v1.23.0/docs/http/http-spans.md
-    public const string AttributeClientAddress = "client.address";
-    public const string AttributeClientPort = "client.port";
-    public const string AttributeHttpRequestMethod = "http.request.method"; // replaces: "http.method" (AttributeHttpMethod)
-    public const string AttributeHttpRequestMethodOriginal = "http.request.method_original";
-    public const string AttributeHttpResponseStatusCode = "http.response.status_code"; // replaces: "http.status_code" (AttributeHttpStatusCode)
-    public const string AttributeNetworkProtocolVersion = "network.protocol.version"; // replaces: "http.flavor" (AttributeHttpFlavor)
-    public const string AttributeNetworkProtocolName = "network.protocol.name";
-    public const string AttributeServerAddress = "server.address"; // replaces: "net.host.name" (AttributeNetHostName) and "net.peer.name" (AttributeNetPeerName)
-    public const string AttributeServerPort = "server.port"; // replaces: "net.host.port" (AttributeNetHostPort) and "net.peer.port" (AttributeNetPeerPort)
-    public const string AttributeUrlFull = "url.full"; // replaces: "http.url" (AttributeHttpUrl)
-    public const string AttributeUrlPath = "url.path"; // replaces: "http.target" (AttributeHttpTarget)
-    public const string AttributeUrlQuery = "url.query";
-    public const string AttributeUrlScheme = "url.scheme"; // replaces: "http.scheme" (AttributeHttpScheme)
```

Comment:
There was an early effort (https://github.com/open-telemetry/opentelemetry-dotnet/pull/5432) to sync the shared files between `opentelemetry-dotnet` and `opentelemetry-dotnet-contrib`. We might either need to keep the files in sync or as Reiley recommended move these attributes to actual implementation.

Reply:
I think that this repository needs only small subset of full semantic convetion from contrib repository. We can move to actual implementation, but it could be more error prone (two copies of the same string). I would not expect frequent changes in attributes/resources needed in this repository (most of them already should be stable).